docs/vexim-acl-check-rcpt.conf

```diff
@@ -52,5 +52,16 @@
 #                          and spamassassin='1'}}}{1} {yes}{no}}
 #          !acl          = spf_rcpt_acl
 
+#
+# To use a DNSBL the best would be to set up a caching dns server for you own
+# otherwise you might get rate limited and your requests are not processed
+# 
+# Also disable spamchecking for authenticated users because dial up networks
+# are often marked as bad in case of sending mails
+#
+  accept
+          authenticated = *
+          condition = ${if eq{$authenticated_id}{$sender_address}{yes}{no}}
```

Comment:
Just thinking about it, I have a main address with login (`$authenticated_id`) but I often use a different identities with a specific `$sender_address` (redirection to my main account). This wouldn't work.